orchestration/task_dispatcher.py

```
@@ -31,50 +50,60 @@ def __init__(self, orchestration_dir: str = None):
-        
+
         # Check tmux sessions
         try:
             result = subprocess.run(
-                ["tmux", "list-sessions", "-F", "#{session_name}"], 
+                ["tmux", "list-sessions", "-F", "#{session_name}"],
                 capture_output=True, text=True
             )
             if result.returncode == 0:
                 existing.update(result.stdout.strip().split('\n'))
         except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)